scst-store/amr/graphql-query.hbs.md

```diff
@@ -0,0 +1,261 @@
+# AMR Graphql Querying
+
+This page contains information on how to connect to the GraphQL playground, and how to query each supported data model.
+See [AMR Data Models](data-model-and-concepts.hbs.md) for more information on the AMR data models.
+
+## Connecting to Graphql Playground
+
+The recommended configuration is to enable ingress. The Supply Chain Security Tools for Tanzu – Store and Artifact Metadata Repository (AMR) packages share the same ingress configuration. 
+As such setting `ingress_domain` for the Store will apply the same value for the AMR.
```

Comment:
⛏️"Store" should be renamed to "SCST - Store", which is the official short form.